nailgun/entities.py

```diff
@@ -4005,9 +4006,12 @@ def update(self, fields=None):
 
     def update_payload(self, fields=None):
         """Wrap submitted data within an extra dict."""
-        return {
-            u'organization': super(Organization, self).update_payload(fields)
-        }
+        org_payload = super(Organization, self).update_payload(fields)
+        payload = {u'organization': org_payload}
+        if 'redhat_repository_url' in org_payload:
+            rh_repo_url = org_payload['redhat_repository_url']
```

Comment:
Yes, it filters out the extra param, I'm pretty sure. At least, it was working for me. Either way, I've updated it to `pop`.